doc/guidelines/e2e.md

```diff
@@ -0,0 +1,102 @@
+## How to write tests
+
+### Locator Guidelines
+
+Before developing E2E tests make sure you have read the [locator guidelines](https://github.com/ElektraInitiative/PermaplanT/doc/guidelines/frontend-locators.md).
+
+### General rules
+
+- Be consistent and minimalistic.
+  - Use the same vocabulary as playwright (click, visible, etc.)
+  - Avoid using multiple different verbs for the same actions, keep your vocabulary small and precise.
+    Use Playwrights vocabulary.
+    This means to prefix methods with the actions from playwright (e.g when calling `xyz.click()` from playwright inside that method, name the method `click_xyz()` not a mix of press, click, push etc.)
+  - Don't indent more than one time.
+  - Don't make a complicated call stack higher than two from a page object.
+- Every test should be independent from other tests (concurrency).
+- Name inputs or objects you create SUT (System under Test) so they are clearly marked as test artifacts.
+
+### A typical workflow
+
+The usual workflow consists of three steps:
+
+1. (`/features`) write a feature inside with one or multiple scenarios.
+2. (`/pages`) implement the page object (if it doesn't already exist) in .
+3. (`/steps`) create a function for each scenario step using the page objects.
+
+Lets go over these steps in more detail.
+
+### Writing .feature files
+
+Make sure to have a solid understanding about the Gherkin syntax, so you don't fall into common pitfalls.
+Usually the syntax is not very strict but poor Gherkin will cascade into the later processes of testing and make everything more complicated.
+
+Avoid the following:
```

Comment:
Probably positive guidelines would be (more) helpful, e.g. "Write all steps in third-person point of view." , "Scenarios should be short and sweet." etc. from https://automationpanda.com/2017/01/30/bdd-101-writing-good-gherkin/